### DISCUSSION TITLE
Training  & Test Set Preparation

### DISCUSSION THREAD

**Discussion by technosaby** [technosaby/gsoc2022#23] [General]
Any suggestions about how to prepare a test set to calculate accuracy. Is there any way to use existing mapped data to calculate accuracy ?

**Comment by brucearctor** [comment]
Getting trustworthy labeled data can be difficult.  


I haven't used, and at this time haven't looked closely enough to understand whether would be appropriate/useful --> https://snorkel.ai/  ... but is on my radar re: generating training data [ I think aimed at a different use case, but ??? ]

I've spent $$$$ on 'expert' human labelers, and I've also outsourced much with https://en.wikipedia.org/wiki/Figure_Eight_Inc. and written very complex workflows to utilize https://www.mturk.com/ ...  Though, I don't think we have money to spend on labeling, so hopefully able to discover additional datasets.  

Traditionally, you'd need labeled data for both training, as well as test/evaluating.  I don't know what all options exist -- so that'd be a good area to spend time on.  I imagine, for instance, there are various sound effects sources of data.  Even if some things aren't labeled/setup for ML in the traditional sense, even filenames can act as metadata which could then be transformed/normalized.  

What all datasets can you discover with some searches?

**Comment by technosaby** [comment]
@turnermarkb Do you have comment on this ?
